src/cnd.h

```diff
@@ -0,0 +1,19 @@
+#ifndef RLANG_CND_H
+#define RLANG_CND_H
+
+#include <stdbool.h>
+
+
+void r_inform(const char* fmt, ...);
+void r_warn(const char* fmt, ...);
+void r_abort(const char* fmt, ...);
+
+SEXP interp_str(const char* fmt, ...);
```

Comment:
I wasn't fighting this convention, we can't clutter the global namespace without a consistent organisation of the symbols. I'll review all to-be-exported functions in another PR.